frontend/src/scenes/events/CommunicationDetails.tsx

```diff
@@ -0,0 +1,45 @@
+import { LemonTextArea } from 'lib/lemon-ui/LemonTextArea/LemonTextArea'
+import { LemonSwitch } from 'lib/lemon-ui/LemonSwitch/LemonSwitch'
+import { communicationDetailsLogic } from './CommunicationDetailsLogic'
+import { useActions, useValues } from 'kea'
+import { IconComment, IconMail } from 'lib/lemon-ui/icons/icons'
+import { LemonButton } from 'lib/lemon-ui/LemonButton/LemonButton'
+
+export function CommunicationDetails({ uuid }: { uuid: string }): JSX.Element {
+    const { publicReplyEnabled, noteContent } = useValues(communicationDetailsLogic)
+    const { togglePublicReply, saveNote, setNoteContent } = useActions(communicationDetailsLogic)
+
+    console.log(uuid)
+
+    return (
+        <>
+            <LemonSwitch
```

Comment:
I think we should use LemonSegmentedButton here so someone can pick "email" or "note" without having to think about what the switch status means https://storybook.posthog.net/?path=/docs/lemon-ui-lemon-segmented-button--default